internal/cmd/argus/grafana/public-read-access/disable/disable.go

```diff
@@ -0,0 +1,115 @@
+package disable
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/errors"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/flags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/globalflags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/print"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/argus/client"
+	argusUtils "github.com/stackitcloud/stackit-cli/internal/pkg/services/argus/utils"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/utils"
+
+	"github.com/spf13/cobra"
+	"github.com/stackitcloud/stackit-sdk-go/services/argus"
+)
+
+const (
+	instanceIdFlag = "instance-id"
+)
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+	InstanceId string
+}
+
+func NewCmd(p *print.Printer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "disable",
+		Short: "Disables public read access for Grafana on Argus instances",
+		Long: fmt.Sprintf("%s\n%s",
+			"Disables public read access for Grafana on Argus instances.",
+			"When disabled, a login is required to access the Grafana dashboards without of the instance logging in. Otherwise, anyone can access the dashboards.",
```

Comment:
```suggestion
			"When disabled, a login is required to access the Grafana dashboards without logging in. Otherwise, anyone can access the dashboards.",
```